README.md

```diff
@@ -27,30 +27,24 @@ $ python manage.py runserver
 
 ## Code Style
 
-This codebase uses the PEP8 code style. We enforces this with flake8. We use almost all the rules
-used in the Requests project.
-
-In addition to the standards outlined in PEP 8, we have a few guidelines:
+This codebase uses the PEP8 code style. We enforces this with isort, yapf & flake8.
+In addition to the standards outlined in PEP 8, we have a few guidelines
+(see `setup.cfg` for more info):
 
 * Line-length can exceed 79 characters, to 100, when convenient.
 * Always use single-quoted strings (e.g. 'lego'), unless a single-quote occurs within the string.
 
-Additionally, one of the styles that PEP8 recommends for line continuations completely lacks all
-sense of taste, and is not to be permitted within the Requests codebase:
-
+Format the code with yapf in an editor plugin, or run:
```

Comment:
Is there a decent pycharm plugin for yapf? https://plugins.jetbrains.com/plugin/9705-yapf-pycharm says it needs yapf globally installed, but haven't tested. Could be nice since a lot of people use pycharm, and it's what we recommend to new members.

Reply:
A few links would be nice! :smile: 
Haven't tested pycharm yet, but in vim it just werkz :tm: